app/views/admin/reports/index.html.haml

```diff
@@ -5,31 +5,34 @@
   .filter-subset
     %strong= t('admin.reports.status')
     %ul
+      %li= filter_link_to t('admin.reports.all'), resolved: '-1'
       %li= filter_link_to t('admin.reports.unresolved'), resolved: nil
       %li= filter_link_to t('admin.reports.resolved'), resolved: '1'
   .filter-subset
     %strong= t('admin.reports.target_origin')
     %ul
       %li= filter_link_to t('admin.accounts.location.all'), target_origin: nil
-      %li= filter_link_to t('admin.accounts.location.local'), target_origin: 'local'
-      %li= filter_link_to t('admin.accounts.location.remote'), target_origin: 'remote'
+      %li= filter_link_to t('admin.accounts.location.local'), target_origin: 'local', search_term: nil, search_type: nil
+      %li= filter_link_to t('admin.accounts.location.remote'), target_origin: 'remote', search_term: nil, search_type: nil
 
 = form_with url: admin_reports_url, method: :get, class: :simple_form do |form|
   .fields-group
-    - ReportFilter::KEYS.each do |key|
-      = form.hidden_field key, value: params[key] if params[key].present?
+    = form.hidden_field :resolved, value: params[:resolved] if params[:resolved].present?
 
-    - %i(by_target_domain).each do |key|
-      .input.string.optional
-        = form.text_field key,
-                          value: params[key],
-                          class: 'string optional',
-                          placeholder: I18n.t("admin.reports.#{key}")
```

Comment:
By removing this line, the following localisation keys became unused:

```
+--------+----------------------------------------------+-------------------------------------------------------------+
| Locale | Key                                          | Value                                                       |
+--------+----------------------------------------------+-------------------------------------------------------------+
|   en   | admin.reports.quick_actions_description_html | Take a quick action or scroll down to see reported content: |
|   en   | admin.reports.reopen                         | Reopen report                                               |
|   en   | admin.reports.reported_account               | Reported account                                            |
|   en   | admin.reports.updated_at                     | Updated                                                     |
+--------+----------------------------------------------+-------------------------------------------------------------+
```

I don't know how to resolve this.

Reply:
Have just gone ahead and removed these — hopefully they're not used somehow